### ISSUE TITLE
Update Apache Commons IO library

### ISSUE THREAD

**Issue by IE-smhi** [baltrad/beast#5]
## Details
It has come to our attention that when using BEAST in remote file systems and when using parts of Baltrad separately with a custom-made producer, sporadic logging errors related to file distribution failures are occurring. 

Specifically, users are encountering the following error message:

```
File distribution failed! java.io.IOException: Failed to copy full contents [...]
```

## Root cause:
The error originates from the Apache Commons IO library, which is currently at version 2.4. 
This version has known issues that can lead to sporadic file distribution failures, particularly in networked file systems.

See JIRA tickets: 
* https://issues.apache.org/jira/browse/IO-443
* https://issues.apache.org/jira/browse/IO-544
* https://issues.apache.org/jira/browse/IO-386
* ...

Essentially the most applicable fix that we want is [IO-443](https://issues.apache.org/jira/browse/IO-443).

## TL;DR:

Upgrade Apache Commons IO library version 2.12.0.

**Comment by IE-smhi** [comment]
As of commits https://github.com/baltrad/beast/commit/b8f12f5e095c154268365e7b2cf5a02eecb4ee1a https://github.com/baltrad/beast/commit/a7cf887f337d4057d13bb456e60be6027217a51a the changes this issue required is now in master. 
Deleting branch https://github.com/baltrad/beast/tree/5-update-apache-commons-io-library .

Closing this issue.
